README.md

```diff
@@ -52,7 +55,8 @@ In order to build and run the samples in this repository, you need:
         1. Build your project. You do not need to use the same command prompt for building.
         1. Run the following command: `vcperf /stopnoanalyze MySessionName outputTraceFile.etl`

```

Comment:
For Code Analysis it should be Visual Studio 2022 17.11 or later. You can add a note here. Also, they can run this without an elevated command prompt using vcperf /start /noadmin MySessionName